ci/ci-util.js

```diff
@@ -8,42 +8,67 @@ var semver = require('semver');
 //------------------------------------------------------------------------------
 
 var buildTasksPath = path.join(__dirname, '..', '_build', 'Tasks');
-var packagePath = path.join(__dirname, '..', '_package');
-var tasksLayoutPath = path.join(packagePath, 'tasks-layout');
-var tasksZipPath = path.join(packagePath, 'tasks.zip');
-var hotfixLayoutPath = path.join(packagePath, 'hotfix-layout');
-var milestoneLayoutPath = path.join(packagePath, 'milestone-layout');
-var milestonePackSourcePath = path.join(packagePath, 'milestone-pack-source');
-var milestonePackSourceContentsPath = path.join(packagePath, 'milestone-pack-source', 'contents');
-var milestoneNuspecPath = path.join(packagePath, 'vsts-tasks-milestone.nuspec');
-var restorePath = path.join(packagePath, 'restore');
-var aggregateLayoutPath = path.join(packagePath, 'aggregate-layout');
-var aggregatePackSourcePath = path.join(packagePath, 'aggregate-pack-source');
-var aggregatePackSourceContentsPath = path.join(packagePath, 'aggregate-pack-source', 'contents');
-var aggregatePackSourceContentsZipPath = path.join(packagePath, 'aggregate-pack-source', 'contents', 'Microsoft.TeamFoundation.Build.Tasks.zip');
-var aggregatePackageName = 'Mseng.MS.TF.Build.Tasks';
-var aggregateNuspecPath = path.join(packagePath, 'Mseng.MS.TF.Build.Tasks.nuspec');
-var publishLayoutPath = path.join(packagePath, 'publish-layout');
-var publishPushCmdPath = path.join(packagePath, 'publish-layout', 'push.cmd');
 exports.buildTasksPath = buildTasksPath;
+
+var packagePath = path.join(__dirname, '..', '_package');
```

Comment:
why change the style here, now it requires a lot of scrolling to attempt to understand the paths. 1 out of every 3 lines is interesting now.

Reply:
Prefer the export near the value to stay consistent, can revert.

Reply:
Reverted.